src/OpenTelemetry.Exporter.Jaeger/README.md

```diff
@@ -1,129 +0,0 @@
-# Jaeger Exporter for OpenTelemetry .NET
-
-[![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Exporter.Jaeger.svg)](https://www.nuget.org/packages/OpenTelemetry.Exporter.Jaeger)
```

Comment:
We could consider keeping this folder for some time and redirect users to https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/trace/getting-started-jaeger. That way its easier for users to discover next steps when we deprecate the package.